scripts/react_native_pods.rb

```diff
@@ -141,7 +141,11 @@ def use_flipper!(versions = {}, configurations: ['Debug'])
   use_flipper_pods(versions, :configurations => configurations)
 end
 
-def react_native_post_install(installer, react_native_path = "../node_modules/react-native")
+def react_native_post_install(installer, mac_catalyst_enabled, react_native_path = "../node_modules/react-native")
+  if mac_catalyst_enabled
+    ReactNativePodsUtils.apply_mac_catalyst_patches(installer)
+  end
```

Comment:
[nit] We can use a [conditional modifier](https://riptutorial.com/ruby/example/2878/inline-if-unless) here:

```suggestion
ReactNativePodsUtils.apply_mac_catalyst_patches(installer) if mac_catalyst_enabled
```

Reply:
I assume, the double assert_equal in **third_target.build_configurations.each** block is redundant, right?

Reply:
Yeah, sorry... bad copy-and-paste... 🤦